platform/security/platform-security-core-api/src/test/java/ddf/security/samlp/SamlProtocolTest.java

```diff
@@ -156,6 +165,13 @@ public void testCreateSpMetadata() {
             .get(1)
             .getLocation());
     assertEquals(
+        "paoslocation",
+        entityDescriptor
+            .getSPSSODescriptor(SamlProtocol.SUPPORTED_PROTOCOL)
+            .getAssertionConsumerServices()
+            .get(2)
```

Comment:
It can be brittle to depend on the order of a list this way. Use Hamcrest collection matchers.  Update the other uses of `get()` to reduce technical debt.

I wrote some code that should clean things up a bit: https://gist.github.com/ahoffer/f33d7e9631eb63008d784c18f011a986

Reply:
Updated the SSOServices and AssertionConsumerServices to use streams to be a bit less brittle.